package.json

```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "workspaces": [
+        "./packages/unified-latex-util-pegjs",
```

Comment:
Is there a reason this was added explicitly? Was it not captured by `./workspaces/*`?

Reply:
Yes. For other modules to be compiled (to cjs), pegjs needs to be first compiled. This is why the package is explicitly stated before the wildcard. Unfortunately, I cannot figure out another solution, though I think this is not bad.